.github/workflows/main.yml

```diff
@@ -0,0 +1,38 @@
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+
+jobs:
+  ci:
+    name: CI
+    runs-on: ubuntu-latest
+    steps:
+      - name: Install Dependencies
+        run: sudo apt-get update && sudo apt-get install curl jq -y
+
+      - uses: actions/checkout@master
+        with:
+          fetch-depth: 50
+
+      - name: Install mdbook
+        run: |
+          # Retrieve latest mdbook release.
+          RELEASES="$(curl https://api.github.com/repos/rust-lang/mdbook/releases/latest)"
+          # Filter to just the url for the `unknown-linux-gnu` release.
+          URL=$(echo $RELEASES | jq -r '.assets[] as $in | $in.name | match("unknown-linux-gnu") | $in.browser_download_url')
+          # Download and untar the binary into the path.
+          curl -L $URL | sudo tar xz -C /usr/local/bin
+      - name: Build book
+        run: |
+          mdbook build
+          touch book/.nojekyll
+
+      - name: Deploy to GitHub Pages
+        run: |
+          curl -LsSf https://raw.githubusercontent.com/rust-lang/simpleinfra/master/setup-deploy-keys/src/deploy.rs | rustc - -o /tmp/deploy
+          (cd book/ && /tmp/deploy)
```

Comment:
This is also missing CloudFront invalidation, which is needed for the forge.

Reply:
@pietroalbini I don't know how to add that, is there documentation or an example I can pull from?